packages/bpk-component-select/src/BpkSelect.js

```diff
@@ -59,37 +86,58 @@ const BpkSelect = props => {
   if (className) {
     classNames.push(className);
   }
-
-  return (
+  if (imageContainerClassName) {
+    imageContainerClassNames.push(className);
+  }
+  if (wrapperClassName) {
+    wrapperClassNames.push(className);
+  }
+  const select = (
     <select
       className={classNames.join(' ')}
       aria-invalid={isInvalid}
       {...rest}
     />
   );
+
+  if (image) {
+    return (
+      <div className={wrapperClassNames.join(' ')}>
```

Comment:
Instead of using `wrapperClassNames` you can use Tiago's extensions to `getClassNames` and write this as

```
className={getClassName('bpk-select__wrapper', large && 'bpk-select__wrapper--large', wrapperClassName)}
```
Same goes for the image container